### PR TITLE
Integrate level-js for browser support

### LINKED ISSUE #50: Merge level and level-browserify
The only difference is that `level-browserify` has a `browser.js` which pulls in `level.js`. What do you think of merging them into one, e.g. deprecating `level-browserify` and adding `level.js` to `level`?

It would mean less to maintain and also simple to use `level` and it "just works".

Upcoming version of `level` will bundle `levelup@2`, together with `encoding-down` and have a promise layer as well, for people that prefer that.

cc @Level/owners 

- [ ] Release `level-browserify@2` first so we can get some testing

### PR DESCRIPTION
**Pull request by vweevers**
Closes #50. Tasks left:

- [x] Release and update `level-js`
- [x] Go through readme, change references to LevelDB / `leveldown` where necessary (we can also choose to do this after the prerelease)

### CODE REVIEW

**Comment by vweevers** [comment]
Tested locally in Chrome, IE and Edge.

File: README.md [hunk 1]
```diff
@@ -75,13 +94,11 @@ db.put('name', 'Level', function (err) {
...
 * [<code>db.<b>createValueStream()</b></code>](#createValueStream)
 
-See [`levelup`][levelup] and [`leveldown`][leveldown] for more details.
-
 <a name="ctor"></a>
 ### `db = level(location[, options[, callback]])`
 The main entry point for creating a new `levelup` instance.
 
-- `location` path to the underlying `LevelDB`.
+- `location` is a string pointing to the LevelDB location to be opened or in browsers, the name of the [`IDBDatabase`](https://developer.mozilla.org/en-US/docs/Web/API/IDBDatabase) to be opened.
```
Comment (vweevers) [inline README.md]: Should we change examples to `db('my-db')` rather than `db('./my-db')`?

Comment (ralphtheninja) [reply README.md]: Yes, much cleaner

File: README.md [hunk 2]
```diff
@@ -244,10 +267,13 @@ Clear all queued operations on the current batch, any previous operations will b
 
 The number of queued operations on the current batch.
 
-<b><code>batch.write([callback])</code></b>
+<b><code>batch.write([options][, callback])</code></b>
 
 Commit the queued operations for this batch. All operations not *cleared* will be written to the underlying store atomically, that is, they will either all succeed or fail with no partial commits.
 
+- `options` is passed on to the underlying store.
+- `options.keyEncoding` and `options.valueEncoding` are not supported here.
```
[the remark this answers is not in the document]
Comment (ralphtheninja) [reply README.md]: Maybe do this to level up repo?

[the remark this answers is not in the document]
Comment (vweevers) [reply README.md]: https://github.com/Level/levelup/issues/633

**Pull request merged**